### ISSUE TITLE
When compiling the Molgenis-app in Eclipse the downloading of the maven-metadata.xml times out. 

### ISSUE THREAD

**Issue by JanTimmermans** [molgenis/molgenis#4652]
When compiling the Molgenis-app in Eclipse the downloading of the maven-metadata.xml times out. This results in a long compiling time.

**To reproduce:**
Run the Molgenis-app in Eclipse (settings according to http://molgenis.github.io/develop/start)
**Expected:**
No, or few time-outs
**Result:**
A long compiling time due to loop for downloading the maven-metadata.xml 

```
[INFO] >>> jetty-maven-plugin:9.2.6.v20141205:run (default-cli) > test-compile @ molgenis-app >>>
[INFO] Downloading: https://oss.sonatype.org/content/repositories/snapshots/org/molgenis/molgenis-core/1.18.0-SNAPSHOT/maven-metadata.xml
[INFO] Downloading: http://nexus.cs.toronto.edu/nexus/content/repositories/snapshots/org/molgenis/molgenis-core/1.18.0-SNAPSHOT/maven-metadata.xml
[WARNING] Could not transfer metadata org.molgenis:molgenis-core:1.18.0-SNAPSHOT/maven-metadata.xml from/to uoft-snapshots (http://nexus.cs.toronto.edu/nexus/content/repositories/snapshots/): connect timed out
[WARNING] Failure to transfer org.molgenis:molgenis-core:1.18.0-SNAPSHOT/maven-metadata.xml from http://nexus.cs.toronto.edu/nexus/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of uoft-snapshots has elapsed or updates are forced. Original error: Could not transfer metadata org.molgenis:molgenis-core:1.18.0-SNAPSHOT/maven-metadata.xml from/to uoft-snapshots (http://nexus.cs.toronto.edu/nexus/content/repositories/snapshots/): connect timed out

```

**Comment by Mark-de-Haan** [comment]
Make sure you have imported all the modules into your eclipse workspace.
And what OS are you running?

**Comment by fdlk** [comment]
The 1.18.0-SNAPSHOT releases are not supposed to be in the central maven repositories, as they're snapshot releases. You build them locally and eclipse will install them in your local maven repository.

Have you installed the m2e maven plugin for eclipse?

**Comment by fdlk** [comment]
We have a link to uoft snapshot repository in the molgenis-app pom. Is that really necessary?

**Comment by dennishendriksen** [comment]
Fixed in #4813 available since MOLGENIS 1.21
